src/datasources/exchange-api/entities/exchange-result.entity.ts

```diff
@@ -1,5 +1,5 @@
 export interface ExchangeResult {
   success: boolean;
-  rates?: Record<string, number>;
+  rates?: object;
```

Comment:
Can we leave this as `Record`? We are loosing the type information which is useful when working with this class.

Reply:
I tried, but I wasn't able to keep the `Record` type and make the `JSONSchemaType` at `exchange.result.schema` compatible, there isn't a lot of documentation regarding that integration. I didn't want to get this feature blocked due to this, so maybe could we open an issue to tackle this? What do you think?

Reply:
Commented out in [84c402f](https://github.com/5afe/safe-client-gateway-nest/commit/84c402fcafe4e82ec8817e0092b9fd35427ec1a3).

I plan to give this a second chance when tacking [this](https://github.com/5afe/safe-client-gateway-nest/issues/72) 🙂